__tests__/resolvers/exotics/github-resolver.js

```diff
@@ -27,15 +27,26 @@ test('getGitSSHUrl with no hash', () => {
   expect(gitSSHUrl).toContain('some-user');
 });
 
-test('getGitHTTPUrl should return the correct git github SSH url', () => {
+test('getGitHTTPBaseUrl should return the correct git github HTTP base url', () => {
   const fragment: ExplodedFragment = {
     user: 'foo',
     repo: 'bar',
     hash: '',
   };
 
-  const expected =  'git+ssh://git@github.com/' + fragment.user + '/' + fragment.repo + '.git';
-  expect(GitHubResolver.getGitSSHUrl(fragment)).toBe(expected);
```

Comment:
Previously the `getGitHTTPUrl` test was actaully running `getGitSSHUrl` instead!